test/unit/api/api.test.js

```diff
@@ -372,6 +372,14 @@ describe('API', () => {
     });
 
     describe('undo', () => {
+      beforeEach(() => {
+        process.env.NODE_ENV = 'production';
```

Comment:
we should probably test the logic doesn't get called when in development mode.  It would be much easier to do this if we had the wrapper as we'd only need to test the wrapper itself.